transient_auth/views.py

```diff
@@ -0,0 +1,16 @@
+from django.http import HttpResponse
+from django.views.decorators.http import require_http_methods
+from django.views.decorators.csrf import csrf_exempt
+from .authentication import check_pdf_authenticity, check_html_authenticity
+
+
+@csrf_exempt
+@require_http_methods(['POST'])
+def authenticate(request):
+  url = request.POST.get('url')
+  if url.endswith('.pdf'):
+    pdf = request.FILES['pdf'].file.getvalue()
+    return HttpResponse(check_pdf_authenticity(pdf, url))
+
+  content = request.POST.get('content')
```

Comment:
In case of html, we send textual data, in case of pdfs, we need to send a file. Didn't seem like a problem to me since we have the browser extensions which prepare and send the request. We could use `FormData` in both cases to send path and content.
As we discussed at the time, the prototype only had to support pdf and html. I don't think that validating other binary files should be very similar to how pdfs are handled, so that should be easy to implement. What other files except for `.html` and `.pdf` do we need to handle? I think it's better to address that in a separate PR and archive the old repository.